byudml/imputer/random_sampling_imputer.py

```diff
@@ -96,20 +106,27 @@ def produce(self, *, inputs: Inputs, timeout: float = None, iterations: int = No
                 )
             )
 
+        drop_cols_all_unknown_vals: bool = self.hyperparams[strings.DROP_COLS_ALL_UNKNOWN_VALS_NAME]
+        if drop_cols_all_unknown_vals:
+            columns_to_remove: list = []  # Columns that have no known values in them need to be dropped
```

Comment:
@erikhuck Learning which columns to drop should happen in the fit function. In `produce`, if a column of `inputs` has all missing values, but the training data has know values, we want to impute, not drop the columns.

Reply:
I will take care it. This is just an FYI.